cmd/flawfind/main.go

```diff
@@ -216,7 +217,13 @@ func init() {
 				slog.Error("could not create request", "err", err)
 				return
 			}
-			req.Header.Set("Authorization", "Bearer "+token)
+
+			err = pat.DoCryptoChallenge(token, req)
```

Comment:
I think we can rename this to "SignRequest". I know, DoCryptoChallenge was my Idea, but since we are now using an official RFC, we can specify this a bit more.